tests/unit/AddNameTranslation.spec.ts

```diff
@@ -31,163 +30,190 @@ describe('Add Name Translation component', () => {
     // Init Store
     store.state.stateModel.nameTranslations = []
 
-    wrapperFactory = (propsData) => {
-      return mount(AddNameTranslation, {
+    wrapperFactory = async (propsData: any) => {
+      const wrapper = mount(AddNameTranslation, {
         localVue,
         router,
         store,
         vuetify,
         propsData: { ...propsData }
       })
+      await Vue.nextTick()
+      return wrapper
     }
   })
 
-  it('displays the input field and buttons in the Add Name Translation form', () => {
-    const wrapper = wrapperFactory()
+  it('displays the input field and buttons', async () => {
+    const wrapper = await wrapperFactory()
 
     // Verify input field
-    expect(wrapper.find(addTranslationInput).exists()).toBeTruthy()
+    expect(wrapper.find(nameTranslationInput).exists()).toBeTruthy()
 
-    // Verify Action btns and there default states
+    // Verify Action btns and their default states
     expect(wrapper.find(doneBtn).exists()).toBeTruthy()
-    expect(wrapper.find(doneBtn).attributes('disabled')).toBeTruthy()
```

Comment:
"Truthy" means anything that isn't falsy, so it isn't very specific. Better to check for the value you expect.